gcd/src/main/java/org/spine3/server/storage/datastore/DsAggregateStorage.java

```diff
@@ -55,6 +68,7 @@
     private static final String SNAPSHOT = "SNAPSHOT";
 
     private static final String KIND = AggregateStorageRecord.class.getName();
+    private static final String AGGREGATE_ENTITY_STATUS_KIND = EntityStatus.class.getName();
```

Comment:
This is done to reduce the odds that the framework user will accidentally overlap e.g. `EntityStatus` type, which seems possible for me.